.github/ISSUE_TEMPLATE/pull-request.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Pull Request
+about: Add new Code Examples to Catalog
+title: "[NEW EXAMPLE] <<Language>>"
+labels: ''
+assignees: ''
+
+---
+
+Each example should contain the following metadata. Use comment denotations appropriate for each langague.
+Example:
+
+```
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").a
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+*/
+//snippet-sourcedescription:[CatchDeleteError.java demonstrates how to try to delete a job and catch an error.]
+// snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+// snippet-sourcedescription:[<<FILENAME>> demonstrates how to ...]
+// snippet-service:[<<ADD SHORT SERVICE NAME>>]
+// snippet-sourcesyntax:[<<LANGUAGE>>]
+// snippet-keyword:[<<LANGUAGE>>]
+// snippet-keyword:[<<ADD FULL SERVICE NAME>>]
+// snippet-keyword:[Code Sample]
+// snippet-keyword:[<<API OPERATION>>]
+// snippet-sourcetype:[snippet] or snippet-sourcetype:[full-example]
+// snippet-sourcedate:[YYYY-MM-DD]
+// snippet-sourceauthor:[<<YOUR USERNAME>>]
+// snippet-start:[service.language.unique_name.complete]
+
+<your code>
+
+// snippet-end:[service.language.unique_name.complete]
+```
+
```

Comment:
If necessary, how do I build/compile the example?
If necessary, how do I run the example?
Are there any required inputs?
What output should I expect?
Are there any hard-coded values that might need changing?

Reply:
Should you indicate which metadata bits must have only one instance (such as snippet-sourcedescription and snippet-sourcedate), and which can have multiple? For example, snippet-service should support multiple instances: one for each service demonstrated in a complex example.  How does the submitter know this?